src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -290,6 +291,28 @@ $CURL $CONTEXT_URL/v$selected_tag/one-context-$selected_tag-r1.apk -Lsfo /root/c
 apk add --allow-untrusted /root/context.apk >> /var/log/chroot.log 2>&1
 rm /root/context.apk
 
+rm /dev/random /dev/urandom
+EOC
+)
+    ;;
+*alt*)
+    terminal="/bin/bash"
+    commands=$(cat <<EOC
+export PATH=/sbin:/usr/sbin:/bin:/usr/bin
+
+rm -f /etc/resolv.conf >> /var/log/chroot.log 2>&1
+echo "nameserver $DNS_SERVER" > /etc/resolv.conf
+
+[ ! -e /dev/random ] && mknod -m 666 /dev/random c 1 8  >> /var/log/chroot.log 2>&1
+[ ! -e /dev/urandom ] && mknod -m 666 /dev/urandom c 1 9  >> /var/log/chroot.log 2>&1
+
+apt-get update >> /var/log/chroot.log 2>&1
+apt-get install $PKG_ALT -y >> /var/log/chroot.log 2>&1
+
+$CURL $CONTEXT_URL/v$selected_tag/one-context-$selected_tag-1.alt.noarch.rpm -Lsfo /root/context.rpm >> /var/log/chroot.log 2>&1
```

Comment:
Package release follows other ALT packages, so it won't be `1.alt` as it would be with Red Hat*, but `alt1`. E.g.,
```
one-context-5.12.0-alt1.noarch.rpm
```
https://github.com/OpenNebula/addon-context-linux/blob/master/generate.sh#L71

Reply:
Fixed, thank you!